modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/builder/CronJobSimpleExpressionBuilder.java

```diff
@@ -28,6 +28,7 @@ public CronJobSimpleExpressionBuilder(Time startTime) {
      * Sets interval on which job should be fired.
      *
      * @param repeatIntervalInDays  the interval(in days) between job fires, 0 means everyday
+     *
```

Comment:
new line between description and param is ok, however param, return and throws should be without new line. Follow the javadoc conventation used in mds module. 

This comment regards other changes